tests/flexmock_test.py

```diff
@@ -1064,14 +1119,6 @@ def test_module_level_function_with_kwargs(self):
         flexmock(mod).should_receive("module_level_function").with_args(1, args="expected")
         assert_raises(FlexmockError, module_level_function, 1, args="not expected")
 
-    def test_flexmock_should_support_mocking_old_style_classes_as_functions(self):
```

Comment:
Did you remove old classes support?

Reply:
There are no old style classes in Python 3

Reply:
Ah right we're on this branch :+1: